src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -203,115 +194,145 @@ export function DiscoverLegacy({
+                      <div className="dscDiscover">
+                        <DocTableLegacy
+                          columns={state.columns || []}
+                          indexPattern={indexPattern}
+                          minimumVisibleRows={minimumVisibleRows}
+                          rows={rows}
+                          sort={state.sort || []}
+                          searchDescription={opts.savedSearch.description}
+                          searchTitle={opts.savedSearch.lastSavedTitle}
+                          onAddColumn={addColumn}
+                          onFilter={onAddFilter}
+                          onMoveColumn={onMoveColumn}
+                          onRemoveColumn={onRemoveColumn}
+                          onSort={onSort}
                         />
-                      </h2>
-                      {rows && rows.length && (
-                        <div className="dscDiscover">
-                          <DocTableLegacy
-                            columns={state.columns || []}
-                            indexPattern={indexPattern}
-                            minimumVisibleRows={minimumVisibleRows}
-                            rows={rows}
-                            sort={state.sort || []}
-                            searchDescription={opts.savedSearch.description}
-                            searchTitle={opts.savedSearch.lastSavedTitle}
-                            onAddColumn={addColumn}
-                            onFilter={onAddFilter}
-                            onMoveColumn={onMoveColumn}
-                            onRemoveColumn={onRemoveColumn}
-                            onSort={onSort}
-                          />
-                          <a tabIndex={0} id="discoverBottomMarker">
-                            &#8203;
-                          </a>
-                          {rows.length === opts.sampleSize && (
-                            <div
-                              className="dscTable__footer"
-                              data-test-subj="discoverDocTableFooter"
+                        <span tabIndex={-1} id="discoverBottomMarker">
+                          &#8203;
+                        </span>
+                        {rows.length === opts.sampleSize && (
+                          <div className="dscTable__footer" data-test-subj="discoverDocTableFooter">
+                            <FormattedMessage
+                              id="discover.howToSeeOtherMatchingDocumentsDescription"
+                              defaultMessage="These are the first {sampleSize} documents matching
+                  your search, refine your search to see others."
+                              values={{ sampleSize: opts.sampleSize }}
+                            />
+
+                            <EuiButtonEmpty
+                              onClick={() => {
+                                const skipToBottomBtn = document.getElementById('dscSkipButton');
```

Comment:
Not required at all, but there is a more elegant way to get a reference of the button by [forwarding refs](https://reactjs.org/docs/forwarding-refs.html) on the `SkipBottomButton` component.

Reply:
thx @flash1293, this is good knowledge! However it's not necessary anymore since targeting a different element to set focus to (`main`), as @myasonik suggested